source/_templates/layout.html

```diff
@@ -1,8 +1,17 @@
 {# Import the theme's layout. #}
 {% extends "!layout.html" %}
 
+
 {# Custom CSS overrides #}
 
+{% block extrahead %}
+
+  {{ super() }}
+
+   <script type="text/javascript" src="//script.crazyegg.com/pages/scripts/0088/9671.js" async="async"></script>
```

Comment:
It appears the GA scripts are placed inside the `body` tag. Can we place them inside the `head`?

Reply:
Done